scripts/TestHarness/testers/RavenFramework.py

```diff
@@ -265,10 +265,10 @@ def prepare(self):
       if os.path.exists(filename):
         os.remove(filename)
 
-  def process_results(self, output):
+  def process_results(self, _):
     """
       Check to see if the test has passed.
-      @ In, output, string, output of test.
+      @ In, ignored, string, output of test.
```

Comment:
will it be used in the future?

Reply:
The output is used currently in testers like RavenErrors.  Basically, we need it for the subclasses that use it.